force-app/main/default/lwc/batchProgress/batchProgress.js

```diff
@@ -96,6 +98,13 @@ export default class BatchProgress extends LightningElement {
         }, this.pollingTimeout, self);
     }
 
+    /***
+    * @description Sets the Total Records value based on Processed and Failed
+    */
+    setTotalRecords() {
+        this.batchJob.summary.total = this.batchJob.summary.processed + this.batchJob.summary.failed;
+    }
```

Comment:
I think I'd change this up a tiny bit to add that total property on to the object before assigning it to `this.batchJob`. Here's a suggestion below:

```suggestion
    setTotalRecords(data) {
        if(!isNull(data)) {
           data.summary.total = data.summary.processed + data.summary.failed;
           return data;
        }
        return null;
    }
```

Then with that, we could do something like:

`this.batchJob = setTotalRecords(JSON.parse(data));`